web/vtadmin/src/components/routes/workflow/WorkflowStreams.tsx

```diff
@@ -14,106 +14,19 @@
  * limitations under the License.
  */
 
-import { groupBy, orderBy } from 'lodash-es';
-import React, { useMemo } from 'react';
-import { Link } from 'react-router-dom';
+import React from 'react';
```

Comment:
We should take this opportunity to address the linter warnings below.

Reply:
Oh! Forgot to remove that.